tutorials/2021_microgames.md

```diff
@@ -0,0 +1,13 @@
+---
+layout: page
+title: 2021 Microgames
+---
+
+2021 Microgames
+===============
+
+Welcome to Student Robotics 2021
+
+Here is a link to the [SR2021 Microgames](https://docs.google.com/document/d/1pKYGQo6m1OjOMAJ4-IqPvfbQ71G60QBVo27lDeKToUY/edit)
+
+We recommend you work through them as a team, so you can learn from each other. If you have any issues, just ask us on [Discord](https://discord.com/channels/775497131057741836/777582218662510612) (Your team leader should have the joining instructions).
```

Comment:
Suggest linking to the docs on discord, rather than just bouncing them to a page they can't use without more information anyway:
```suggestion
We recommend you work through them as a team, so you can learn from each other. If you have any issues, just ask us on [Discord](/docs/team_admin/discord).
```